src/compute/src/render/reduce.rs

```diff
@@ -1381,15 +1447,13 @@ where
                     row_builder.packer().push(value);
                     (key, row_builder.clone())
                 })
-                .map(|k| (k, ()))
-                .mz_arrange::<KeySpine<(Row, Row), _, _>>(
-                    "Arranged Accumulable Distinct [val: empty]",
-                )
-                .mz_reduce_abelian::<_, KeySpine<_, _, _>>(
+                .map(move |(k, v)| (key_val_pairer.merge(k, v), ()))
```

Comment:
Can this `map` be inlined into the previous `map`?

Reply:
Done in https://github.com/MaterializeInc/materialize/pull/24375, rebased here now.